src/bun.js/test/jest.zig

```diff
@@ -430,6 +493,213 @@ pub const TestRunner = struct {
 pub const Jest = struct {
     pub var runner: ?*TestRunner = null;
 
+    pub const Snapshots = struct {
+        pub var snapshot_count: usize = 0;
+        pub var update_snapshots: bool = false;
+        pub var initialized: bool = false;
+        pub var file_header = "// Bun Snapshot v1, https://goo.gl/fbAQLP\n";
+
+        pub var snapshot_dir: std.fs.Dir = undefined;
```

Comment:
none of these should be global variables

all should be part of the `TestRunner`. Probably this `snapshots` struct should have all these values